backend/src/routes/api/health/index.ts

```diff
@@ -0,0 +1,15 @@
+import { FastifyInstance, FastifyReply, FastifyRequest } from 'fastify';
+import { health } from './healthUtils';
+
+export default async (fastify: FastifyInstance): Promise<void> => {
+  fastify.get('/', async (request: FastifyRequest, reply: FastifyReply) => {
+    return health(fastify)
+      .then((res) => {
+        return res;
+      })
+      .catch((e) => {
+        fastify.log.error(`Failed to get status, ${e.response?.data?.message || e.message}}`);
+        reply.send(e.response?.data?.message || e.message);
+      });
```

Comment:
Small nit, because you have fallbacks that will work... throwing an error of `createCustomError` will just get you `e.message` as a value. `e.response.data.message` is the kube calls way of giving back an error.

`e.response.data` is a k8s status object.

(also as noted in a DM, you muted the status code by not using it with reply).